app/lib/dttp/code_sets/institutions.rb

```diff
@@ -5,193 +5,193 @@ module CodeSets
+        "University College Birmingham" => { entity_id: "fa70f34a-2887-e711-80d8-005056ac45bb" },
+        "University College Cardiff" => { entity_id: "" }, # Not on the list
+        "University College London" => { entity_id: "a27af34a-2887-e711-80d8-005056ac45bb" },
+        "University for the Creative Arts" => { entity_id: "354b9247-7042-e811-80ff-3863bb3640b8" },
+        "University of Bedfordshire" => { entity_id: "fc70f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Bolton" => { entity_id: "dfdb7129-7042-e811-80ff-3863bb3640b8" },
+        "University of Bradford" => { entity_id: "7fed6e2f-7042-e811-80ff-3863bb3640b8" },
+        "University of Chester" => { entity_id: "0871f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Cumbria" => { entity_id: "0c71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Derby" => { entity_id: "0e71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Durham" => { entity_id: "1071f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Essex" => { entity_id: "5aeb7735-7042-e811-80ff-3863bb3640b8" },
+        "University of Glamorgan" => { entity_id: "" }, # Not on the list
+        "University of Gloucestershire" => { entity_id: "1871f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Hertfordshire" => { entity_id: "1c71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Huddersfield" => { entity_id: "1e71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Kent" => { entity_id: "99eb7735-7042-e811-80ff-3863bb3640b8" },
+        "University of Lancaster" => { entity_id: "a4eb7735-7042-e811-80ff-3863bb3640b8" },
+        "University of London" => { entity_id: "5243a44d-7042-e811-80ff-3863bb3640b8" }, # Mistmached name "University of London (Institutes and activities)"
+        "University of Manchester" => { entity_id: "5b43a44d-7042-e811-80ff-3863bb3640b8" },
+        "University of Newcastle-upon-Tyne" => { entity_id: "ea70f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Northumbria at Newcastle" => { entity_id: "2a71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Nottingham" => { entity_id: "2c71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Plymouth" => { entity_id: "3071f34a-2887-e711-80d8-005056ac45bb" },
+        "University of South Wales" => { entity_id: "8723a753-7042-e811-80ff-3863bb3640b8" },
+        "University of Southampton" => { entity_id: "4f5b7f3b-7042-e811-80ff-3863bb3640b8" },
+        "University of Surrey" => { entity_id: "58228041-7042-e811-80ff-3863bb3640b8" },
+        "University of Sussex" => { entity_id: "3e71f34a-2887-e711-80d8-005056ac45bb" },
+        "University of Ulster" => { entity_id: "a823a753-7042-e811-80ff-3863bb3640b8" },
+        "University of Wales Trinity Saint David" => { entity_id: "b123a753-7042-e811-80ff-3863bb3640b8" },
+        "University of Wales, Cardiff" => { entity_id: "" }, # Not on the list
```

Comment:
Have we got a plan for these missing ones?

Reply:
All entity ID's are populated now.